examples/webgl_animation_animation-pointer.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - animation - keyframes</title>
```

Comment:
Do you mind renaming the file to `webgl_gltf_animation_pointer?

The title can then be `three.js webgl - gltf - animation - pointer`.

Reply:
The example isn't related to webgl tho - the same code should work with webgpu and we're basically just enhancing the ability of PropertyBindings here